.gitignore

```diff
@@ -0,0 +1,34 @@
+profile
+*.moved-aside
+DerivedData
+.idea/
+
+# xcode noise
+build/*
+*.perspective
+*.perspectivev3
+*.pbxuser
+*.mode1
+*.mode2v3
+*.mode1v3
+xcuserdata
+Snapshots
+test-reports/
+*.xccheckout
+*.xcconfig
+*.xcscmblueprint
+
+# osx noise
+.DS_Store
+Thumbs.db
+
+# Carthage
+.carthage_cache.yml
+Carthage/
+
+Frameworks/
+
+.swiftlint-report.json
+
+
+fastlane/report.xml
```

Comment:
Just checking, isn't there a fastlane/output or some other fastlane not wanted files?